docs/concepts/pdp/configuration.mdx

```diff
@@ -39,15 +39,29 @@ The URL of the Permit.io control plane.
 
 Default: `None`
 
-PDP debug mode configuration.
-There available configurations are:
-- `PDP_DEBUG=None` - The default behavior, will use the configuration set by Permit.io Cloud.
+Enables debug mode, which injects debug attributes to OPA policy queries.
+The available configurations are:
+- `PDP_DEBUG=None` - The default behavior - will use the configuration set by Permit.io Cloud.
```

Comment:
I think there is a warning that this has a performance impact in the debug-mode page, but worth verifying this